parameter_tools/src/test/resources/ExampleJsonSyncClass.json

```diff
@@ -1,5 +1,5 @@
 {
-    "testText": "Hello World",
+    "testcText": "Hello World",
```

Comment:
Should this say testText instead of testcText? Looks like it's still testText in the other file.

Reply:
That was just a typo